AdminUi/test/AdminUi.Tests.Integration/Models/HttpResponse.cs

```diff
@@ -10,3 +10,11 @@ public class HttpResponse<T>
     public string? ContentType { get; set; }
     public string? RawContent { get; set; }
 }
+
+public class HttpResponse
+{
+    public HttpStatusCode StatusCode { get; set; }
+    public bool IsSuccessStatusCode { get; set; }
+    public string? RawContent { get; set; }
```

Comment:
Why is there a RawContent? This shouldn't ever be set, right?

Reply:
Yes, I'll remove it.